__tests__/app.test.js

```diff
@@ -114,4 +115,49 @@ describe('GET /api/articles', () => {
             }
         });
     });
+});
+
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('returns array of comment objects with the given article ID and the correct properties', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments
+
+            expect(Array.isArray(comments)).toBe(true);
+
+            expect(comments).toBeSortedBy('created_at', {descending: true});
+
+            for(comment of comments) {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
Instead of separately checking the article id below to be 1, you can have the value 1 here instead of expect.any(Number)